tests/migration_test.go

```diff
@@ -3399,9 +3405,16 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					tests.StartVirtualMachine(vm_noevict)
 
 					// Get VMIs
-					vmi_evict1, err = virtClient.VirtualMachineInstance(vmi_evict1.Namespace).Get(vmi_evict1.Name, &metav1.GetOptions{})
-					vmi_evict2, err = virtClient.VirtualMachineInstance(vmi_evict1.Namespace).Get(vmi_evict2.Name, &metav1.GetOptions{})
-					vmi_noevict, err = virtClient.VirtualMachineInstance(vmi_evict1.Namespace).Get(vmi_noevict.Name, &metav1.GetOptions{})
+					vmi_evict1, err = virtClient.VirtualMachineInstance(vm_evict1.Namespace).Get(vm_evict1.Name, &metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
```

Comment:
This change does not seem to be related to this commit.
Same for the other error and NodeName checking below.

Reply:
It's hardening the test, it really does not need another commit I think.

Reply:
It is an unexpected change, I prefer you move it to a separate commit and explain it is for hardening/fixing.

Reply:
I agree - the test looks better, and is a lot more deterministic from the added expectations, but this change is unrelated to "using `libvmi.NewAlpine()`"